aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -63,6 +64,12 @@
   private final FileIO fileIO;
   private final LockManager lockManager;
 
+  // Attempt to load versionId if available on the path
+  private static final DynMethods.UnboundMethod LOAD_VERSION_ID = DynMethods.builder("versionId")
+      .hiddenImpl("software.amazon.awssdk.services.glue.model.UpdateTableRequest$Builder", String.class)
+      .orNoop()
+      .build();
```

Comment:
It's a little unclear what this method is doing. It looks like this sets a requirement that the table has not changed since the given version ID? If so, then I think the comment below could be more clear.

Reply:
Yes, correct.

```
/**
 * Sets the value of the VersionId property for this object.
 *
 * @param versionId
 *        The new value for the VersionId property for this object.
 * @return Returns a reference to this object so that method calls can be chained together.
 */
Builder versionId(String versionId);
```